README.md

```diff
@@ -41,7 +41,7 @@ Hosting (production):
 
 ## Systems overview
 
-For a more complete overview see: #9
+For a more complete overview see: https://github.com/alveusgg/alveusgg/issues/9
```

Comment:
```suggestion
For a more complete overview see: [#9](https://github.com/alveusgg/alveusgg/issues/9)
```
For cleanliness, I think it'd be better to keep the text as `#9` still?

Reply:
It renders as `#9` on github, but it wouldn't hurt to be portable.